### PR TITLE
Working through a good first issue

### PR DESCRIPTION
**Pull request by ATorrise**
**What It Does**
Corrects #2138 

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @ATorrise and @jace-roell!

**Review by traeok — APPROVED** [review]
LGTM, thanks @jace-roell and @ATorrise!

File: packages/imperative/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Modify error text in SyntaxValidator.invalidOptionError. [#2138](https://github.com/zowe/zowe-cli/issues/2138)
```
Comment (zFernand0) [inline packages/imperative/CHANGELOG.md:7]: Sorry for being picky, but whenever there are text-only changes I think we go with a patch/fix category.

The original issue if labeled as an enhancement so I can understand why we say it here too.

But curious about when we backport this to the `master` branch...
Do we plan to publish this as a `release-patch` or `release-minor` ? 🤔

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 
Left a small question, but since this PR is against `next` we don't have to address it here 😋

File: packages/imperative/CHANGELOG.md (hunk 1 above)
Comment (ATorrise) [reply packages/imperative/CHANGELOG.md:7]: release-patch!

**Pull request merged**